src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,7 @@
+package duke;
```

Comment:
Probably putting it into an exception package would improve like understanding of your code especially when gradle is implemented.